src/lib/locale/constructor.js

```diff
@@ -1,5 +1,7 @@
+import { set } from './set';
+
 export function Locale(config) {
     if (config != null) {
-        this.set(config);
+        set.call(this, config);
```

Comment:
This got reconfigured because the prototype's copy of `set` is wrapped for immutability, which is decidedly unhelpful inside a constructor.

Note that we hadn't explicitly discussed making Locale objects immutable, but it seemed like the obvious thing to do.